doc-content/enterprise-only/installation/securing-passwords-eap-proc.adoc

```diff
@@ -0,0 +1,92 @@
+[id='securing-passwords-eap-proc_{context}']
+
+= Securing passwords with a keystore
+
+You can use a keystore to encrypt passwords that are used for communication between {CENTRAL} and {KIE_SERVER}. You should encrypt both controller and {KIE_SERVER} passwords. If {CENTRAL} and {KIE_SERVER} are deployed to different application servers then both application servers should use the keystore.
+
+Use Java Cryptography Extension KeyStore (JCEKS) for your keystore because it supports symmetric keys. Use KeyTool, which is part of the JDK installation, to create a new JCEKS.
+
+[NOTE]
+====
+If {KIE_SERVER} is not configured with JCEKS, {KIE_SERVER} passwords are stored in system properties in plain text form.
+====
+
+.Prerequisites
+
+* {KIE_SERVER} is installed in {EAP}.
+* Java 8 or higher is installed.
+
+.Procedure
+. In the {EAP} home directory, enter the following command to create a {KIE_SERVER} user with the `kie-server` role and specify a password. In the following example, replace `<USER>` and `<PASSWORD>` with the user and password of your choice.
++
+[source,bash]
+----
+$<EAP_HOME>/add-user.sh -a -e -u <USER> -p "<PASSWORD>" -g kie-server
```

Comment:
I think there's no quotes around **<PASSWORD>** in this case. See <USER> for example. How you have it elsewhere, like **password="<PASSWORD>"** is correct however.

Also, I suggest "replace `<USER_NAME>` ..." and update the code snippet with "-u <USER_NAME>" because that's what you do in the others. Either way, note the issue that you say "replace <USER>" in all versions of this section but all  others say "<USER_NAME>" in the snippet itself.

Reply:
I made the USER_NAME change, but I'll check with Karel about the command.